src/app.ts

```diff
@@ -106,8 +109,7 @@ app.get("/auth/callback", async (c) => {
   const code = c.req.query("code");
   const state = c.req.query("state");
   const error = c.req.query("error");
-
-
+  const clientId = c.req.query("client_id");
```

Comment:
This doesn't seem to make sense, `client_id` is not an expected parameter here, so it'll just be always undefined.